### ISSUE TITLE
Errors with `Dropdown` with custom slots and Typescript

### ISSUE THREAD

**Issue by Lucide** [carbon-design-system/carbon-components-svelte#1202]
I can't get the [slotted](https://carbon-components-svelte.onrender.com/components/Dropdown#custom-slot) Dropdown to work with Typescript:
![image](https://user-images.githubusercontent.com/33134209/159753933-1ae6db24-7794-48f6-9cb9-0ddfc5a8af73.png)

```svelte
<script lang="ts">
    import { Dropdown } from "carbon-components-svelte";
    let items = [
        { id: "0", text: "Slack" },
        { id: "1", text: "Email" },
        { id: "2", text: "Fax" },
    ];
</script>

<div>
    <Dropdown titleText="Contact" selectedId="0" {items} let:item let:index>
        <div>
            <strong>{item.text}</strong>
        </div>
        <div>
            id: {item.id} - index:
            {index}
        </div>
    </Dropdown>
</div>

<style lang="scss">
    :global(.bx--list-box__menu-item, .bx--list-box__menu-item__option) {
        height: auto;
    }
</style>
```

In the console I get this warning on every refresh: `<Dropdown> received an unexpected slot "default".`
The rendered output shows the Dropdown with the standard UI. Do I need to do something special to make it work with ts?

**Comment by metonym** [comment]
What version of `carbon-components-svelte` are you using?

**Comment by Lucide** [comment]
I'm using svelte latest `3.46.4` and carbon-components-svelte latest `0.62.2`.

**Comment by metonym** [comment]
This what I see using Svelte/TypeScript in my IDE: 

<img width="791" alt="Screen Shot 2022-03-23 at 1 15 24 PM" src="https://user-images.githubusercontent.com/10718366/159787923-bad03f06-f588-4963-94bc-ea7fe1f2a806.png">

Can you try restarting the Svelte Language Server in your IDE?

**Comment by Lucide** [comment]
Very strange, in a desperate attempt to reproduce the problem, I ended up recreating the entire project from scratch inside a test folder (incremental file copy-paste, no rewriting).
Now I have two ~identical folders, one that exhibits the issue and one that doesn't...
Tried:
* renaming root folder
* clean npm reinstall
* tinkering with every file in the distro

At a closer inspection, this error might be different from the typing issue I mentioned earlier, which occurs nonetheless.
(an entire package/library is shown as having type `any`). I'm using VSCode, with the official Svelte extension.
